src/mumble/AudioOutputUser.h

```diff
@@ -14,14 +14,20 @@ class AudioOutputUser : public QObject {
 		Q_DISABLE_COPY(AudioOutputUser)
 	protected:
 		unsigned int iBufferSize;
+
+		/// Used to resize the buffer.
+		/// WARNING:
+		///          Audio callback is a dedicated place that can be executed
+		///          in a special thread or interrupt handler. Allocating
+		///          memory will probably crash the program!
```

Comment:
I'd vote in for completely removing the ability to resize the buffer.

Reply:
Then I have to implement this construct-time initialization of the buffer for all audio backends. 🤔️  That would be a hard job since I'm only capable of testing these things with CoreAudio (since I'm a Mac user).

Reply:
I guess we'll find testers for all major platforms :)